docs/bib/use_citations.bib

```diff
@@ -1,3 +1,15 @@
+% 2021-06-03
+@article{ATLAS:SUSY-3L-compressed-combination,
```

Comment:
@kratsg I just threw in this more descriptive identifier over `Aad:2021ajl` for the time being as there isn't an entry for it in the `ATLAS.bib` yet. If you know what it will be or want a different identifier feel free to change it. :+1: